_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -176,9 +222,25 @@ No heading.
 
 #### Inapplicable example 2
 
-Heading that is neither visible to users, nor included in the accessibility tree.
+The heading is neither visible to users, nor included in the accessibility tree.
 
 ```html
 <h1 style="display: none;">Opening hours</h1>
 <p>We are open Monday through Friday from 10 to 16</p>
 ```
+
+#### Inapplicable example 3
+
+The heading marked up with `h` element does not contain visible content or has an accessible name that does not only consist of [Unicode separator characters](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153).
+
+```html
+<h1></h1>
+```
+
+#### Inapplicable example 4
+
+The heading marked up with `role="heading"` does not contain visible content or has an accessible name that does not only consist of [Unicode separator characters](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153).
+
+```html
+<h1 aria-label=" "></h1>
```

Comment:
In the markup, the heading does not role="heading". I suggest you use the span element in stead of the h element to mirror the other test cases, such as failed example 3.

Reply:
Well spotted! Corrected now...